### PR TITLE
feat(preset): Add oracle database jdbc and nls monorepo groups

### PR DESCRIPTION
**Pull request by jadefr**
## Changes

- Add com.oracle.database.jdbc and com.oracle.database.nls into same groupId, as they follow the same version.

- I was not able to find the repos' github, but it can be seen in maven repository that the versions thereof are the same:

 https://mvnrepository.com/artifact/com.oracle.database.jdbc/ojdbc8
 https://mvnrepository.com/artifact/com.oracle.database.nls/orai18n

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: data/kubernetes-api.json5 [hunk 1]
```diff
@@ -148,8 +148,8 @@
   ImageRepository: ['image.toolkit.fluxcd.io/v1beta2'],
   ImageUpdateAutomation: [
     'image.toolkit.fluxcd.io/v1beta1',
-    'image.toolkit.fluxcd.io/v1beta2'
-    ],
+    'image.toolkit.fluxcd.io/v1beta2',
```
Comment (rarkins) [inline data/kubernetes-api.json5:151]: Revert this unrelated change

**Review by rarkins — CHANGES_REQUESTED** [review]

File: data/kubernetes-api.json5 (hunk 1 above)
Comment (jadefr) [reply data/kubernetes-api.json5]: Done, thank you.

**Review by rarkins — APPROVED** [review]
